examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/ExampleGenerator.java

```diff
@@ -85,11 +86,28 @@ public void generate() throws IOException, TemplateException {
             Map<String, String> dataModel = new Yaml().loadAs(input, Map.class);
             String feature = dataModel.get("feature");
             String framework = dataModel.get("framework");
+            generateDir(dataModel, new ExampleScenarioFactory(feature, framework).getJavaClassPathSet(), JAVA_CLASS_PATH);
+            generateDir(dataModel, new ExampleScenarioFactory(feature, framework).getResourcePathSet(), RESOURCES_PATH);
             generateFile(dataModel, new ExampleScenarioFactory(feature, framework).getJavaClassTemplateMap(), JAVA_CLASS_PATH);
             generateFile(dataModel, new ExampleScenarioFactory(feature, framework).getResourceTemplateMap(), RESOURCES_PATH);
+            String outputPath = generatePath(dataModel, OUTPUT_PATH);
+            processFile(dataModel, "pom.ftl", outputPath + "pom.xml");
         }
     }
     
+    private void generateDir(final Map<String, String> dataModel, final Set<String> javaPath, final String outputRelativePath) throws IOException, TemplateException {
+        for (String each : javaPath) {
+            String outputPath = generatePath(dataModel, OUTPUT_PATH + outputRelativePath + "/" + each);
+            processDir(outputPath);
+        }
+    }
+    
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    private void processDir(String path) {
+        File file = new File(path);
+        file.mkdirs();
+    }
```

Comment:
How about inline this method into generateDir?
This method is too short and can merge into 1 line.

Reply:
ok.